osu.Game.Tests/Visual/Gameplay/TestSceneStoryboardWithOutro.cs

```diff
@@ -133,6 +133,16 @@ public void TestStoryboardRewind()
             AddUntilStep("storyboard ends", () => Player.GameplayClockContainer.GameplayClock.CurrentTime >= currentStoryboardDuration);
         }
 
+        [Test]
+        public void TestPerformExitNoOutro()
+        {
+            CreateTest(null);
+            AddStep("disable storyboard", () => LocalConfig.SetValue(OsuSetting.ShowStoryboard, false));
+            AddUntilStep("completion set by processor", () => Player.ScoreProcessor.HasCompleted.Value);
+            AddStep("exit via pause", () => Player.ExitViaPause());
+            AddAssert("score not shown", () => !Player.IsScoreShown);
```

Comment:
I think you should test the essential behaviour of this instead? i.e. assert that the player screen has been exited from and was not blocked (probably gonna be `Stack.CurrentScreen == null`).